tree/master/cloud/src/solution/Deployment/Scripts/Deploy-AzureResourceGroup.ps1

```diff
@@ -37,6 +42,7 @@ Set-Content -Path $TemplateParametersFile -Value (ConvertTo-Json -InputObject $p
 Import-Module Azure -ErrorAction SilentlyContinue
 #Add-AzureAccount
 $subsc = Login-AzureRmAccount
+$global:TenantName = $subsc.Context.Tenant.Domain
```

Comment:
global variable for tenant url which will be used in matter web app settings deployment